addons/mail/static/src/messaging/entity/composer/composer.js

```diff
@@ -188,6 +190,39 @@ function ComposerFactory({ Entity }) {
             return htmlString;
         }
 
+        /**
+         * @private
+         */
```

Comment:
missing `@param` and `@returns`, and maybe also a description considering the length of the method